datafn/cli/src/migrations/render-postgres.ts

```
@@ -0,0 +1,80 @@
+        // For Phase 25 scope (TV-MIG-001 is about addField), we implement basic scaffolds.
+        lines.push(`-- Create table ${change.resource}`);
+        lines.push(`CREATE TABLE "${change.resource}" (id TEXT PRIMARY KEY);`);
+        // Real implementation would iterate columns from schema lookup
+        break;
+
+      case "removeResource":
+        lines.push(`DROP TABLE IF EXISTS "${change.resource}";`);
+        break;
+
+      case "addField":
+        const nullClause = change.required ? "NOT NULL" : "NULL";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)